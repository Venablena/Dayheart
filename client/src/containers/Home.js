import React from 'react';
import '../styles/index.css'
import { Link } from 'react-router-dom'
import { Button, Form, Segment, Container } from 'semantic-ui-react'

const Home = ({}) => (
  <main>
    <div className="App-header">
      <img src='./DayHeart_logo_192.png' className="App-logo" alt="logo" />
      <p>
        Find a childcare provider near you.
      </p>
    </div>
    <Container>
      <Form>
        <Form.Field>
          <input placeholder="Enter zip code" />
        </Form.Field>
        <Form.Group inline>
         <label>Radius</label>
            <Form.Radio value='5 miles' label='5 miles'/>
            <Form.Radio value='10 miles' label='10 miles' />
            <Form.Radio value='15 miles' label='15 miles' />
            <Form.Radio value='20 miles' label='20 miles' />
        </Form.Group>
      </Form>
        <Button className='orange'>
          <Link to="/search">Search</Link>
        </Button>
        <Button className='orange'>
          <Link to="/login">Log in</Link>
        </Button>
    </Container>
  </main>
);

export default Home;
