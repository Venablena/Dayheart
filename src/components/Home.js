import React from 'react';
import { Link } from 'react-router-dom';
import { Button, Input, Divider, Popup } from 'semantic-ui-react';
import Header from './Header';
import SearchInput from './SearchInput';

const Home = () => {
  return (
    <main className= 'centered'>
      <Header />
      <div className= 'wrapper'>
        <h4> Find a child care provider near you</h4>
        { <SearchInput /> }
        <Divider hidden />
        <Link to="/map">
          <Button className='orange' content="Search" />
        </Link>
        <Divider hidden />
        <Link to="/login">
          <Button className='blue'>Log in</Button>
        </Link>
      </div>
    </main>
  )
};

export default Home;
