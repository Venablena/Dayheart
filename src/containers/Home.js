import React from 'react';
import '../styles/index.css'
import { Link } from 'react-router-dom'
import { Button, Form, Container } from 'semantic-ui-react'
import Header from '../components/Header'

const Home = ({}) => (
  <main className= 'centered'>
    <Header />
    <div className= 'wrapper'>
      <Form>
        <Form.Field>
        <label> Find a child care provider near you</label>
          <Form.Input placeholder="Enter zip code" />
        </Form.Field>
        <Form.Group inline>
         <label>Radius</label>
            <Form.Radio value='5 miles' label='5 miles'/>
            <Form.Radio value='10 miles' label='10 miles' />
            <Form.Radio value='15 miles' label='15 miles' />
            <Form.Radio value='20 miles' label='20 miles' />
        </Form.Group>
      </Form>
      <Button className='orange'>
        <Link to="/map">Search</Link>
      </Button>
      <div className= 'buffer'></div>
      <Button className='blue'>
        <Link to="/login">Log in</Link>
      </Button>
    </div>
  </main>
);

export default Home;
