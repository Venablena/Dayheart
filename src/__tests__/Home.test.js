import React from 'react';
import Home from '../components/Home';
describe('Home', () => {
  it('renders without crashing', () => {
    const component = shallow(<Home />);
    expect(component).toMatchSnapshot();
  });

});
