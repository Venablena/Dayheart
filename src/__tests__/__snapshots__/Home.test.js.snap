// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders without crashing 1`] = `
<main
  className="centered"
>
  <Header />
  <div
    className="wrapper"
  >
    <h4>
       Find a child care provider near you
    </h4>
    <Popup
      content="Currently only zip code 98108 has data"
      hideOnScroll={true}
      on="focus"
      position="right center"
      trigger={
        <Input
          icon="search"
          placeholder="Enter zip code"
          transparent={true}
          type="text"
        />
      }
    />
    <Divider
      hidden={true}
    />
    <Link
      replace={false}
      to="/map"
    >
      <Button
        as="button"
        className="orange"
        content="Search"
      />
    </Link>
    <Divider
      hidden={true}
    />
    <Link
      replace={false}
      to="/login"
    >
      <Button
        as="button"
        className="blue"
      >
        Log in
      </Button>
    </Link>
  </div>
</main>
`;
